migrations/2019-08-08-191129_add_users_public/up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE users ADD COLUMN public BOOLEAN NOT NULL DEFAULT 'f';
+UPDATE users
+SET public = 't'
+WHERE
+    id IN (
```

Comment:
It seems like we can probably just use this select instead of adding this column to the table, or perhaps make a `public_users` view with this `WHERE` clause? Was there another reason for making this a persistent column?

Reply:
The reason I made this a persistent column is so we can use row-level security to filter the rows. The comments on #630 explain how I arrived at this design. The most important reason is [this comment by Carol](https://github.com/rust-lang/crates.io/issues/630#issuecomment-510642673).